pkg/controllers/resources/pods/translate/hostpath.go

```diff
@@ -75,41 +76,54 @@ func (t *translator) rewriteHostPaths(pPod *corev1.Pod) {
 					pPod.Spec.Volumes[i].HostPath.Path = t.virtualPodLogsPath
 
 					t.log.Debugf("adding original hostPath to relevant containers")
-					pPod = t.addPhysicalPathToVolumesAndCorrectContainers(
-						volume.Name,
-						volume.HostPath.Type,
-						PodLoggingHostPath,
-						PhysicalPodLogVolumeMountPath,
-						podLogMountPath,
-						pPod,
-					)
+
+					if t.skipPhysicalMounts {
+						pPod = t.ensureHostToContainerPropagation(volume.Name, pPod)
+					} else {
+						pPod = t.addPhysicalPathToVolumesAndCorrectContainers(
+							volume.Name,
+							volume.HostPath.Type,
+							PodLoggingHostPath,
+							PhysicalPodLogVolumeMountPath,
+							podLogMountPath,
+							pPod,
+						)
+					}
 				}
 
-				if volume.HostPath.Path == KubeletPodPath &&
+				if strings.TrimSuffix(volume.HostPath.Path, "/") == KubeletPodPath &&
```

Comment:
This is a bug fix. If you don't merge the PR, at least get this fix into the codebase.

Reply:
There are probably other places that have this issue.